emsdk

```diff
@@ -158,9 +159,12 @@ def cmake_generator_prefix():
   elif CMAKE_GENERATOR == 'MinGW Makefiles': return '_mingw'
   return '' # Unix Makefiles and Visual Studio 2013 do not specify a path prefix for backwards path compatibility
 
+def log_message(msg, end='\n'):
+  sys.stderr.write(msg + end)
```

Comment:
This changes all printing to go to `stderr` instead of `stdout`? We certainly don't want that. If you want unbuffered output, you can run the script with `python -u emsdk.py` or set `PYTHONUNBUFFERED` environment variable. See https://stackoverflow.com/questions/107705/disable-output-buffering

(If we diddwant to wrap printing for some reason, it would be better to assign over the `print` function instead of change `print` to `log_message` everywhere)